### DISCUSSION TITLE
Is GH Discussions any good?

### DISCUSSION THREAD

**Discussion by cueckoo** [cue-lang/cue#385] [General] [answered]
_Originally opened by @mvdan in https://github.com/cuelang/cue/discussions/385_

Input welcome.

**Comment by cueckoo** [comment, marked as answer]
_Original reply by @myitcv in https://github.com/cuelang/cue/discussions/385#discussioncomment-14833_

Looks good to me.

**Reply by cueckoo** [reply]
_Original reply by @mvdan in https://github.com/cuelang/cue/discussions/385#discussioncomment-14833_

Marked as an answer, which I think is good for visibility?

**Comment by cueckoo** [comment]
_Original reply by @mpvl in https://github.com/cuelang/cue/discussions/385#discussioncomment-14838_

Looks great, I'll leave this test for now and remove once we a few discussions going.

**Comment by cueckoo** [comment]
_Original reply by @mvdan in https://github.com/cuelang/cue/discussions/385#discussioncomment-14839_

Oh, neat, I can leave emojis on comments too.

**Comment by cueckoo** [comment]
_Original reply by @verdverm in https://github.com/cuelang/cue/discussions/385#discussioncomment-14881_

This seems like a great way to preserve the important conversations that get lost on Slack. If they added chatroom like channels here, would we start leaving Slack?

**Reply by cueckoo** [reply]
_Original reply by @mpvl in https://github.com/cuelang/cue/discussions/385#discussioncomment-14881_

Yes, I think that for many of the discussions this would be a great forum. Anything where it's worth preserving content.

We are considering leaving Slack for Discord for remaining discussions. Discord has a lower barrier to entry (has a great web client) and better communication means for an unpaid setup. Also MD rendering is superior. At the same time, it has no threading. This is great, as it will put more social pressure to move "discussions" to Discussions.

**Reply by cueckoo** [reply]
_Original reply by @verdverm in https://github.com/cuelang/cue/discussions/385#discussioncomment-14881_

I like Gitter more too, great integrations with GitHub

**Comment by cueckoo** [comment]
_Original reply by @proppy in https://github.com/cuelang/cue/discussions/385#discussioncomment-15564_

This also look like a nice way prevent `issues` from being filed with general usage questions.

**Reply by cueckoo** [reply]
_Original reply by @proppy in https://github.com/cuelang/cue/discussions/385#discussioncomment-15564_

I wonder if there are tools to migrate content between the two namespaces? ex:
- move an issue to a discussion
- file an issue from a discussion

**Reply by cueckoo** [reply]
_Original reply by @myitcv in https://github.com/cuelang/cue/discussions/385#discussioncomment-15564_

> This also look like a nice way prevent `issues` from being filed with general usage questions.

That's the idea!

> * move an issue to a discussion
> * file an issue from a discussion

For repository owners/maintainers etc this already exists in a limited form, and the GitHub team are looking to make it even better.

**Comment by cdhunt** [comment]
Any particular reason why Discussions instead of [Stack Overflow](https://stackoverflow.com/questions/tagged/cuelang)?

Asking for both professional and personal curiosity. It may even make sense to include CUE under https://stackoverflow.com/collectives/go.
